### ISSUE TITLE
Client console app: SyncAgent exits program

### ISSUE THREAD

**Issue by Thaval** [Mimetis/Dotmim.Sync#133]
Hi,

i'm trying to implement the library into my current project, but as soon as the client hits the point "agent.SynchronizeAsync()", the client shuts down without any error and exception.
I followed the example from the web page: https://mimetis.github.io/Dotmim.Sync/Web

The client is an usual .NET Core console project and a lot of PDBs are missing like:
"C:\Program Files\dotnet\shared\Microsoft.NETCore.App\2.1.7\System.Diagnostics.DiagnosticSource.dll"

Actually, I don't wanna have another web app just to be able to synchronize. Can't I simply use a console program for the client? 

(I tried adding Core App with nuget, but then the project didn't start anymore)

**Comment by Thaval** [comment]
[MinimalDotmimSyncExample.zip](https://github.com/Mimetis/Dotmim.Sync/files/2809457/MinimalDotmimSyncExample.zip)
Here is a minimal example.

**Comment by Mimetis** [comment]
Two points:

Your call to your `async` method is not `await`-ed.

you should do a 
``` csharp
await TestDotmim();
```

But because you are in the `main` method, you can't mark this method `async`.   
That's why you should call your method like this:

``` csharp
TestDotmim().GetAwaiter().GetResult();

Console.ReadLine();
```

Second point, about your endpoint. You only need 1 endpoint for the whole database to sync.  
That's why I suggest you to create a more generic endpoint, like `api/sync` or `api/database` or whatever you want :)

**Comment by Thaval** [comment]
Good point with `GetAwaiter()`. I got to admit, I'm not well suited yet for PP with C#.

Alright, got you. Thanks for you quick response :)
